integrationTests/testProcessorNode.go

```diff
@@ -1399,6 +1406,114 @@ func (tpn *TestProcessorNode) initMetaInnerProcessors() {
 	)
 }
 
+func (tpn *TestProcessorNode) initNewSystemSCs() {
+	systemVM, _ := tpn.VMContainer.Get(factory.SystemVirtualMachine)
+	err := tpn.initDelegationManager(systemVM)
+	log.LogIfError(err)
+
+	_, err = tpn.AccntState.Commit()
+	log.LogIfError(err)
+}
+
+//TODO remove functions initDelegationManager, processSCOutputAccounts, getUserAccount when TODO from L1368 is done
```

Comment:
"TODO from method initMetaInnerProcessors" instead L1368, as this line is already L1370 :)

Reply:
yes - this could be moved inside initMetaInnerProcessors.

Reply:
done